builds/ci/mqtt.yaml

```diff
@@ -0,0 +1,25 @@
+trigger: none
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+jobs:
+  ################################################################################
+  - job: compliance_amd64
+    ################################################################################
+    displayName: Compliance amd64
+    # dependsOn: check_run_pipeline
+    # condition: eq(dependencies.check_run_pipeline.outputs['check_files.RUN_PIPELINE'], 'true')
+    pool:
+      vmImage: "ubuntu-16.04"
```

Comment:
```suggestion
      vmImage: "ubuntu-latest"
```
?

Reply:
Any issues with `ubuntu-latest`? I see different vmImage value for mqtt-perf and this pipelines

Reply:
It breaks on latest. The checkin tests use the same version